nixos/tests/installer.nix

```diff
@@ -416,13 +481,14 @@ let
           virtualisation.rootDevice = "/dev/vdb";
           virtualisation.bootLoaderDevice = "/dev/vda";
           virtualisation.qemu.diskInterface = "virtio";
+          virtualisation.qemu.options = [
+            "-chardev socket,id=chrtpm,path=/tmp/swtpm-sock"
+            "-tpmdev emulator,id=tpm0,chardev=chrtpm"
+            "-device tpm-tis,tpmdev=tpm0"
+          ];
 
-          # We don't want to have any networking in the guest whatsoever.
-          # Also, if any vlans are enabled, the guest will reboot
-          # (with a different configuration for legacy reasons),
-          # and spend 5 minutes waiting for the vlan interface to show up
-          # (which will never happen).
-          virtualisation.vlans = [];
+          # We don't want to have any networking in the guest appart for the clevis tests.
```

Comment:
A lot of context was removed from this comment. Shouldn't that remain?

Reply:
I can put the context back, but it feels that the comment is wrong because we were able to go around that issue for our test

Reply:
Ok, but don't leave behind the old setting without the comment. Either the comment needs to stay or the setting needs to go.

Reply:
Oh I see. We want `vlans = []` just to avoid any unnecessary network interactions with the installer, but we *have* to have it for clevis. And the part about it not working after reboot appears to be no longer true. So the part of the comment that you left is still accurate and justifies setting `vlans = []`, but the other part is inaccurate? Is that right?

In any case, `appart for` should probably say `apart from` :P

Reply:
Yeah that's right :) Will correct!